ui/00_message/common/spot_name.xml

```diff
@@ -2117,6 +2156,7 @@
   <message>
     <key>SPOT_NAME_949</key>
     <original>水底の古道</original>
+    <translation>Watersdeep Old Road</translation>
```

Comment:
Oh, I didn't mean to use "dry riverbed". I was just saying if that is the case, "Watersdeep Old Road" doesn't make sense.